x-pack/plugins/observability_onboarding/server/routes/custom_logs/route.ts

```diff
@@ -248,11 +242,115 @@ const deleteStatesRoute = createObservabilityOnboardingServerRoute({
   },
 });
 
+const stepProgressResetRoute = createObservabilityOnboardingServerRoute({
```

Comment:
When is this `reset` being used?
Are we planning to reuse apiKeys?

Reply:
This is only being used for development, so i can clean up all the saved objects as i develop. It can be safely removed at any time.

Reply:
OK all dev routes have been removed